<!DOCTYPE html>
<!--[if IE 8]> <html lang="en" class="ie8"> <![endif]-->  
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->  
<!--[if !IE]><!--> <html lang="en"> <!--<![endif]-->  
<head>
    <title>PRIDE ProteoGenomics Pipeline</title>
    <!-- Meta -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Responsive HTML5 Resume/CV Template for Developers">
    <meta name="author" content="Xiaoying Riley at 3rd Wave Media">    
    <link rel="shortcut icon" href="favicon.ico">  
    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,400italic,300italic,300,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
    <!-- Global CSS -->
    <link rel="stylesheet" href="assets/plugins/bootstrap/css/bootstrap.min.css">   
    <!-- Plugins CSS -->
    <link rel="stylesheet" href="assets/plugins/font-awesome/css/font-awesome.css">
    
    <!-- Theme CSS -->  
    <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head> 

<body>
    <div class="wrapper">
        <div class="sidebar-wrapper">
            <div class="profile-container">
                <img class="img-circle" src="assets/images/profile.jpg" alt="" />
                <h1 class="name">PRIDE</h1>
                <h3 class="tagline">Protein Identification Database</h3>
            </div><!--//profile-container-->
            
            <div class="contact-container container-block">
                <ul class="list-unstyled contact-list">
                    <li class="email"><i class="fa fa-envelope"></i><a href="mailto: pride-support@ebi.ac.uk">pride-support@ebi.ac.uk</a></li>
                    <li class="website"><i class="fa fa-globe"></i><a href="http://www.ebi.ac.uk/pride/archive" target="_blank">PRIDE Archive</a></li>
                </ul>
            </div>
            <div class="contact-container container-block">
                <h4 class="section-title">Content</h4>
                <ul class="list-unstyled contact-list">
                    <li class="active"><i class="fa fa-angle-double-right"></i><a href="#proteogenomics">ProteoGenomics Pipeline</a></li>
                    <li><i class="fa fa-angle-double-right"></i><a href="#cluster-section">PRIDE Cluster</a></li>
                    <li><i class="fa fa-angle-double-right"></i><a href="#pogo-section">Genome Coordinates Mapping</a></li>
                </ul>
            </div>
        </div><!--//sidebar-wrapper-->
        
        <div class="main-wrapper">
            
            <section class="section summary-section">
                <h2 class="section-title"><i class="fa fa-book"></i>Description</h2>
                <div    class="summary">
                    <p>Mass spectrometry (MS) based proteomics and next-generation sequencing technologies have improved our understanding of the crosstalk between genome, transcriptome, and proteome
                       and contribute to a better understanding of the variations between healthy and disease states. Substantial advances in MS technologies enable more complete identification
                       and quantification of proteomes, making these data more comparable with transcriptomics. However, most of the omics studies are performed in independent groups and most
                       of the repositories and resources only stores data for a specific omics type: genomics or proteomics. Recently, integrative analysis of quantitative measurements from
                       genomic and proteomic studies have identified novel insights into gene expression regulation, cell signaling, and disease.
                    </p>
                    <p>The <strong>PRIDE Cluster Proteogenomics Pipeline</strong> has the goal of mapping the <a href="http://www.ebi.ac.uk/cluster/">PRIDE Cluster peptide evidences</a> into to ENSEMBL genomes
                        coordinates (<strong>Figure 1</strong>). In the flowing sections, we will explain in details all the steps involve in the ProteoGenomics Mapping Pipeline.
                    </p>
                    <img src="assets/images/cluster-proteogenomics.png" alt="" /></br></br>
                    <p>
                        <strong>Figure 1</strong>: The PRIDE Cluster Proteogenomics Pipeline enables the representation of the <strong>Complete Peptidome Space </strong> of PRIDE database into the ENSEMBL/UCSC Browsers. After each release of
                        <a href="http://www.ebi.ac.uk/pride/cluster">PRIDE Cluster</a> each peptide is map to their corresponding genome coordinates. Finally,
                        a <a href="https://genome.ucsc.edu/goldenpath/help/hgTrackHubHelp.html">trackhub</a> is generated and published in
                        the <a href="https://www.trackhubregistry.org/">ENSEMBL TrackHub Registry</a>
                    </p>
                </div><!--//summary-->
            </section><!--//section-->
            
            <section class="section summary-section">
                <h2 class="section-title" id="proteogenomics"><i class="fa fa-sitemap"></i>PRIDE Cluster Proteogenomics Pipeline</h2>
                <p>
                    The process to map the different MS-based proteomics evidences to the corresponding genome coordinates can be classified in multiple steps:
                    (i) quality assessment of the PRIDE Cluster evidences (ii) Mapping of the corresponding high-quality evidences to genome coordinates; (iii) Generation and publication of the resulted
                    trackhubs in a public Repository (<strong>Figure 2</strong>).
                </p>
                <img src="assets/images/cluster-details.png"/></br></br>

                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title" id="cluster-section">PRIDE Cluster Peptide Evidences</h3>
                        </div><!--//upper-row-->
                        <div class="company">PRIDE Cluster</div>
                    </div><!--//meta-->
                    <div class="details" style="margin-bottom: 2em;">

                        <p>Each Complete Submission in PRIDE is imported into PRIDE Cluster Pipeline using stringent 1% FDR thershold at PSM level.  In order to assess the
                        quality of the corresponding PSMs/Peptides, the <a href="https://github.com/mpc-bioinformatics/pia">Protein Inference Toolkit (PIA)</a> has been used <a href="#pia">[1]</a>. Then, all the resulted identified and
                        un-identified spectra is clustered using the popular framework for big data <a href="http://hadoop.apache.org/">Hadoop</a>. In summary, a spectrum cluster is a group of Mass Spec spectra that share the same characteristics,
                        namely the shapes of their peaks. The spectra that are similar are grouped together in the PRIDE Cluster pipeline <a href="#cluster">[2]</a>.
                        </p>
                        <img src="assets/images/cluster.png"/></br></br>

                    </div><!--//details-->
                </div><!--//item-->
                
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title" id="pogo-section">Mapping the peptide Evidences to Genome Coordinates</h3>
                        </div><!--//upper-row-->
                        <div class="company">PoGo Mapping Tool</div>
                    </div><!--//meta-->
                    <div class="details"  style="margin-bottom: 2em;">
                        <p>The <a href="https://github.com/PRIDE-Toolsuite/PoGo/">PoGo tool</a> is used to map the resulted list of Peptides to the corresponding Genome Coordinates <a href="#pogo">[3]</a>. In summary, PoGo leverages the annotated protein coding sequences (CDS)
                            together with a reference protein sequence database (protein-DB) to map peptides to their genomic loci (<strong>Figure 3</strong>).
                        </p>
                        <img src="assets/images/pogo.png">
                        <br>
                        <br>
                        <p>
                            <strong>Figure 4</strong>: (A) Transcript annotation (GTF) and translated protein sequences (FASTA) form the reference input for PoGo.
                            In addition, the pogo input file containing the peptide sequences, experiment and the quantitative information should be provided.
                            (B) Annotated protein coding transcripts in GTF format and respective translated protein sequences in FASTA format are integrated by
                            PoGo through intermediate coordinates (turquoise), representing the exonic structure of the transcript within the protein. (C) Peptides,
                            identified through searching mass spectrometry data against the protein sequence database, are mapped against the proteins. (D) Example mappings
                            of PoGo for the overlapping repeat peptide VPEPGCTKVPEPGCTK in a genome browser (0 mismatches). Application of PoGo allowing for up to two mismatches
                            results in identification of two additional repeats (1 and 2 mismatches, red boxes). The additional mappings of the initial peptide sequence
                            were validated through peptides of the exact sequence identified in the same mass spectrometry experiment (validation). Leucine (L) and isoleucine
                            (I) are substituted through their common single-letter code ‘‘J.’’ (E) Comparison of different peptide-to-genome mapping tools with regard to
                            reference sequence type, integration into frameworks, support of online and offline genome browsers (blue). Additional features (orange)
                            indicate the superior performance of PoGo over other tools.
                        </p>
                    </div><!--//details-->
                </div><!--//item-->
                
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">TrackHub Creation</h3>
                        </div><!--//upper-row-->
                        <div class="company">TrackHub creator</div>
                    </div><!--//meta-->
                    <div class="details">
                        <p><a href="https://github.com/Proteogenomics/trackhub-creator">The Proteogenomics trackhub creator</a>  provides a set of functionalities to convert the list of peptides evidences
                            into public <strong>trackhub</strong> into the ENSEMBL trackhub registry. Amount other functionalities it enables to download the latest releases of ENSEMBL protein
                            and coordinates data files for specific taxonomies. </p>
                    </div><!--//details-->
                </div><!--//item-->
                
            </section><!--//section-->
            

            <section class="section projects-section">
                <h2 class="section-title"><i class="fa fa-angle-double-right"></i>References</h2>
                <div class="item">
                    <span class="project-title" id="pia">
                        [1] - Uszkoreit J, Maerkens A, Perez-Riverol Y, Meyer HE, Marcus K, Stephan C, Kohlbacher O, Eisenacher M. PIA: An intuitive protein inference engine with a
                        web-based user interface. Journal of proteome research. 2015 Jun 10;14(7):2988-97. <a href="http://pubs.acs.org/doi/abs/10.1021/acs.jproteome.5b00121">Original Publication</a>
                    </span>
                </div><!--//item-->
                <div class="item">
                    <span class="project-title" id="cluster">
                        [2] - Griss J, Perez-Riverol Y, Lewis S, Tabb DL, Dianes JA, del-Toro N, Rurik M, Walzer M, Kohlbacher O, Hermjakob H, Wang R. Recognizing millions of consistently
                        unidentified spectra across hundreds of shotgun proteomics datasets. Nature methods. 2016 Aug 1;13(8):651-6. <a href="http://europepmc.org/backend/ptpmcrender.fcgi?accid=PMC4968634&blobtype=pdf">Original Publication</a>
                    </span>
                </div><!--//item-->
                <div class="item">
                    <span class="project-title" id="pogo">
                        [3] - Schlaffner CN, Pirklbauer GJ, Bender A, Choudhary JS. Fast, Quantitative and Variant Enabled Mapping of Peptides to Genomes. Cell systems. 2017 Aug 23;5(2):152-6.
                        <a href="http://www.sciencedirect.com/science/article/pii/S2405471217303332">Publication</a>
                    </span>
                </div><!--//item-->
            </section>

            <section class="section projects-section">
                <h2 class="section-title"><i class="fa fa-quote-left"></i>Cite Us: </h2>
                <div class="intro">
                    <p>If you use some specific parts of these pipeline, please cite us: </p>
                </div><!--//intro-->
                <div class="item">
                    <span class="project-title">
                        Griss J, Perez-Riverol Y, Lewis S, Tabb DL, Dianes JA, del-Toro N, Rurik M, Walzer M, Kohlbacher O, Hermjakob H, Wang R. Recognizing millions of consistently
                        unidentified spectra across hundreds of shotgun proteomics datasets. Nature methods. 2016 Aug 1;13(8):651-6. <a href="http://europepmc.org/backend/ptpmcrender.fcgi?accid=PMC4968634&blobtype=pdf">Publication</a>
                    </span>
                </div><!--//item-->
            </section><!--//section-->
            
        </div><!--//main-body-->
    </div>
 
    <footer class="footer">
        <div class="text-center">
                <!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can check out other license options via our website: themes.3rdwavemedia.com */-->
            <small class="copyright">Designed <i class="fa fa-laptop"></i> by PRIDE Team</small>
        </div><!--//container-->
    </footer><!--//footer-->
 
    <!-- Javascript -->          
    <script type="text/javascript" src="assets/plugins/jquery-1.11.3.min.js"></script>
    <script type="text/javascript" src="assets/plugins/bootstrap/js/bootstrap.min.js"></script>    
    <!-- custom js -->
    <script type="text/javascript" src="assets/js/main.js"></script>            
</body>
</html> 

